tcnapi/exile/gate/v2/public.proto

```diff
@@ -1356,6 +1365,23 @@ message UnassignAgentSkillRequest {
 }
 message UnassignAgentSkillResponse {}
 
+message CreateRecordingLabelRequest {
+  int64 call_sid = 1;
+  CallType call_type = 2;
+  string key = 3;
+  string value = 4;
+}
+message CreateRecordingLabelResponse {}
+
+message ListRecordingSegmentsRequest {
+  // the recording we are fetching segments for
+  string recording_name = 1;
```

Comment:
there is no recording_name in our api maybe recording_id ?
but this is about searching records so I question the whole payload

Reply:
I'm trying to match what is returned from our api on SearchVoiceRecordings method: https://github.com/tcncloud/exileapi/blob/master/tcnapi/exile/gate/v2/public.proto#L1376